spec/models/king_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+RSpec.describe King, type: :model do
+	 describe 'valid_move?' do
+ 		 it 'should allow a vertical-up move' do
+  			 game = FactoryGirl.create(:game)
+  			 king = King.new(game: game, current_row_index: 5, current_column_index: 5)
+  			 expect(king.valid_move?(6, 5)).to eq true
+  		end
```

Comment:
indentation here is strange, I think rubocop should be auto fixing it.

want 2 spaces :)